contrib/k8s/k8s-unmanaged.sh

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 
 ALL_CEPS=$(kubectl get cep --all-namespaces -o json | jq -r '.items[].metadata | .namespace + "/" + .name')
-ALL_PODS=$(kubectl get pods --all-namespaces -o json | jq -r '.items[].metadata | .namespace + "/" + .name')
+ALL_PODS=$(kubectl get pods --all-namespaces -o json | jq -r '.items[] | select(.spec.hostNetwork==true | not) | .metadata | .namespace + "/" + .name')
```

Comment:
Thanks! 🚀 

I would suggest to add a print statement below (L21) along the lines of "skipping host networked pods".